From d580c708ff6abc75c41cdf71f7b9dc816a4c78da Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Tue, 25 Sep 2018 13:51:22 -0700
Subject: [PATCH 03/13] DO-NOT-UPSTREAM: Disable -Wduplicate-decl-specifier for
 Clang

Clang currently warns on 'const typeof'. This will be fixed upstream
shortly; in the meantime, disable this warning.

GitHub issue: https://github.com/ClangBuiltLinux/linux/issues/52
LLVM bug: https://bugs.llvm.org/show_bug.cgi?id=32985
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index e186d868f866..135acbdf4af9 100644
--- a/Makefile
+++ b/Makefile
@@ -709,6 +709,7 @@ KBUILD_CFLAGS += $(call cc-disable-warning, gnu)
 KBUILD_CFLAGS += $(call cc-disable-warning, address-of-packed-member)
 # Quiet clang warning: comparison of unsigned expression < 0 is always false
 KBUILD_CFLAGS += $(call cc-disable-warning, tautological-compare)
+KBUILD_CFLAGS += $(call cc-disable-warning, duplicate-decl-specifier)
 # CLANG uses a _MergedGlobals as optimization, but this breaks modpost, as the
 # source of a reference will be _MergedGlobals and not on of the whitelisted names.
 # See modpost pattern 2
-- 
2.19.0

